buildtools/scripts/DeployArtifacts.groovy

```diff
@@ -64,6 +64,8 @@ private OptionAccessor parseOptions(arguments)
 def deployJars()
 {
     deployToRemote(ceRepoUrl, ceRepoId, 'org.mule', 'mule', version, 'pom')
+    deployToRemote(ceRepoUrl, ceRepoId, 'org.mule', 'mule-extensions', version, 'pom')
+    deployToRemote(ceRepoUrl, ceRepoId, 'org.mule.modules', 'mule-module-validation', version, 'jar')
```

Comment:
Is this line needed, since you are deploying all org.mule.modules artifacts in the 'for each' below? Aren't you deploying twice?